benchmark_results_data/evm_benchmarks.csv

```diff
@@ -1,108 +1,108 @@
 engine, test_name, total_time, gas_used
```

Comment:
Also, since you are only changing script I think we shouldn't include benchmarks results?

Reply:
I've removed the unrelated files.